test/Microsoft.Azure.WebJobs.Host.EndToEndTests/TestExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Host.EndToEndTests
+{
+    static class TestExtensions
+    {
+        public static XStorageAccount GetStorageAccount(this IHost host)
+        {
+            var provider = host.Services.GetRequiredService<StorageAccountProvider>(); // $$$ ok?
```

Comment:
?

Reply:
Just moved from somewhere else.  We still need to chase down the tail of $$$ todo markers.